composer.json

```diff
@@ -7,7 +7,7 @@
     "require": {
         "php": "^7.2",
 
-        "sylius/sylius": "^1.4.0",
+        "sylius/sylius": "~1.3",
```

Comment:
```suggestion
        "sylius/sylius": "^1.3.0",
```

Reply:
I believe this will restrict it to ´>= 1.3.0 < 1.4.0´

Reply:
Hehe, in fact `~1.3` and `^1.3.0^ are equivalent, however, I like `^1.3.0` better :D